src/ConnectedMode/QualityProfiles/QualityProfileDownloader.cs

```diff
@@ -87,18 +85,22 @@ public async Task<bool> UpdateAsync(BoundSonarQubeProject boundProject, IProgres
         {
             var isChanged = false;
 
+            LogWithBindingPrefix(QualityProfilesStrings.UpdatingQualityProfiles);
```

Comment:
Only two new messages, but it shows the check has started and the number of out of date QPs found.